src/parse.py

```diff
@@ -40,6 +40,25 @@
     # Regardless of the above case, here's how the file name should terminate
     '(\s|$|:)+'
 )))
+MASTER_REGEX_WITH_SPACES = re.compile(''.join((
+    #begin the capture
+    '(',
+    #a leading / for absolute dirs if its there
+    '\/?',
+    # now we look at directories, where we allow either normal chars
+    # or a single whitespace char followed by legit chars
+    '(([a-z.A-Z0-9\-_]|\s[a-zA-Z0-9\-_])+\/)+',
```

Comment:
Hey that's my fault, I read the comment but I didn't scroll up again and look at the tests as closely as I should have.

PR incoming to update the comment to explain that's inner matching.